.circleci/config.yml

```diff
@@ -333,7 +352,7 @@ workflows:
           name: publish-18-run-stack
           image_file: pack-18-run.tar
           image_tag: heroku/pack:18
-          image_tag_alias: heroku/pack:18
+          image_tag_alias: $REGISTRY_HOST/s/heroku/cnb/heroku:18
```

Comment:
Should this be `cnb/pack:18`? This isn't exactly `heroku:18`; it's got a few layers on top...

Reply:
Or maybe call it `heroku/cnb/heroku-run:18` so that it's disambiguated from `heroku/heroku:18` and hints at it's purpose in CNB land?

Reply:
yeah, I always disliked `pack:18` naming. My thought was that since this is under the `cnb` umbrella repository we could ditch the `pack` in favor of `heroku`. We could eventually have `cnb/bionic` and `cnb/functions:20` and whatnot.

Reply:
I'm sort of partial to only having `-build` in the tag. Since, ultimately, customer workloads run on the `cnb/heroku:20` image. It's the most "public" version and I like not having the prefix. I could see us wanting to reference that image name if we allow custom images that derive from that image for rebase-ability or something. But I don't feel too strongly, we can always move things around.

Reply:
I'm 👍 on the new image changes:
* `heroku/cnb/heroku-18:build` - Build
* `heroku/cnb/heroku-18:run` - Run
* `heroku/cnb/heroku-18:builder` - CNB Builder